.github/DISCUSSION_TEMPLATE/ideas-general-discussion.yml

```diff
@@ -0,0 +1,19 @@
+body:
```

Comment:
The filename of the discussion forms must match the category names in the discussion tab on the repository.

This means that if you change the name of the discussion, you should also rename the corresponding template file.